### PR TITLE
NM-34 team wrapper menu

### PR DESCRIPTION
**Pull request by marek-palenik**

### CODE REVIEW

File: src/components/teamwrappermenu/TeamWrapperMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+export interface Props {
+  team:Team,
+  items: TeamWrapperMenuItem[]
+}
+
+const TeamWrapperMenu = (props: Props) => {
+
+  const [anchorEl, setAnchorEl] = React.useState(null);
```
Comment (dullus) [inline src/components/teamwrappermenu/TeamWrapperMenu.tsx]: If you need state then component should be class not function.

File: src/components/teamwrappermenu/TeamWrapperMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  const openMenu = (event: any) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const closeMenu = () => {
+    setAnchorEl(null);
+  };
+
+  const closeMenuWithCallback = (callback: (team: Team) => void, team: Team) => {
+    return () => {
```
Comment (dullus) [inline src/components/teamwrappermenu/TeamWrapperMenu.tsx]: 😳

File: src/components/teamwrappermenu/TeamWrapperMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from './TeamWrapperMenu.module.css';
+import {Menu, MenuItem} from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import {Team} from "../teamwrapper/TeamWrapper";
+
+export interface TeamWrapperMenuItem {
+  label: string;
+  clickHandler: (team:Team) => void;
```
Comment (dullus) [inline src/components/teamwrappermenu/TeamWrapperMenu.tsx]: Is that really necessary to have separate clickHandler for each team ?

Comment (marek-palenik) [reply src/components/teamwrappermenu/TeamWrapperMenu.tsx]: is there any other better way to pass team argument to parent component? for usage like in teamwrappermenu.stories.tsx

File: src/components/teamwrappermenu/TeamWrapperMenu.tsx (hunk 1 above)
Comment (marek-palenik) [reply src/components/teamwrappermenu/TeamWrapperMenu.tsx]: i copied code from material-ui samples

File: src/components/teamwrappermenu/TeamWrapperMenu.tsx (hunk 3 above)
Comment (dullus) [reply src/components/teamwrappermenu/TeamWrapperMenu.tsx]: communication between components will be handled by observing shared redux store

Comment (marek-palenik) [reply src/components/teamwrappermenu/TeamWrapperMenu.tsx]: yup, but for now i tried to keep it simple and stupid, as we are not using redux much right now

**Pull request merged**